File: finding-seams/kotlin/src/test/kotlin/findingseams/SpeakerFeedbackTest.kt
package findingseams

import findingseams.speakerfeedback.EvaluateSpeakerResult
import findingseams.speakerfeedback.SpeakerFeedbackService
import org.junit.jupiter.api.Disabled
import org.junit.jupiter.api.Test
import kotlin.test.assertEquals

//@Disabled("Exercise: finding seams — enable when working on speaker feedback")
class SpeakerFeedbackTest {
    @Test
    fun `calculates average score`() {
        val speakerFeedback = SpeakerFeedbackService()

        val result = speakerFeedback.evaluateSpeaker()

        assertEquals(EvaluateSpeakerResult("OK", 3.6), result)
    }
}

File: finding-seams/csharp/FindingSeams.Tests/SpeakerFeedbackTest.cs
using FindingSeams.SpeakerFeedback;
using Xunit;

namespace FindingSeams.Tests;

public class SpeakerFeedbackTest
{
    [Fact(Skip = "Exercise: finding seams — enable when working on speaker feedback")]
    public void Calculates_average_score()
    {
        var speakerFeedback = new SpeakerFeedbackService();

        var result = speakerFeedback.EvaluateSpeaker();

        Assert.Equal(new EvaluateSpeakerResult("OK", 3.6), result);
    }
}
